.woodpecker/.feature.yml

```diff
@@ -5,6 +5,6 @@ pipeline:
       repo: "${CI_REPO_OWNER%%-vlaanderen}/${CI_REPO_NAME}"
       tags: "feature-${CI_COMMIT_BRANCH##feature/}"
     secrets: [docker_username, docker_password]
-    when:
-      event: push
-      branch: feature/*
+when:
```

Comment:
Is indentation important here? I suspect so since it's different in other services: https://github.com/kanselarij-vlaanderen/agenda-approve-service/tree/master/.woodpecker